refinery/ui/source/js/commons/services/group-member.js

```diff
@@ -5,17 +5,14 @@ angular
   .factory('groupMemberService', ['$resource', 'settings',
     function ($resource, settings) {
       return $resource(
-        settings.appRoot + settings.refineryApiV2 + '/groups/:uuid/members/:id/',
+        settings.appRoot + settings.refineryApiV2 + '/groups/:uuid/members/',
```

Comment:
Simply: `'/groups/:uuid/'` since it's the group resource that is getting updated.